api/src/pcapi/core/finance/api.py

```diff
@@ -2001,7 +1986,7 @@ def _get_cashflows_by_bank_accounts(batch: models.CashflowBatch, only_debit_note
         )
     ).filter(models.Cashflow.batchId == batch.id)
 
-    query = query.filter(models.Cashflow.amount > 0) if only_debit_notes else query.filter(models.Cashflow.amount < 0)
+    query = query.filter(models.Cashflow.amount > 0) if only_debit_notes else query.filter(models.Cashflow.amount < 1)
```

Comment:
Est-ce que ça ne devrait pas être 

```suggestion
    query = query.filter(models.Cashflow.amount > 0) if only_debit_notes else query.filter(models.Cashflow.amount <= 0)
```

du coup ?

Reply:
c'est exactement ça oui `(amount <= 0) ⇔ (amount < 1)` mais je peux changer si ça peu améliorer la lisibilité

Reply:
Pardon j'avais oublié que les montants étaient en centimes, du coup ça revient effectivement au même

Reply:
deal 🤝